state-chain/pallets/cf-vaults/src/tests.rs

```diff
@@ -1036,193 +49,18 @@ fn key_unavailable_on_activate_returns_governance_event() {
-		do_full_key_rotation();
-	});
-}
-
-#[test]
-fn can_recover_from_abort_vault_rotation_after_key_verification() {
-	new_test_ext().execute_with(|| {
-		let rotation_epoch = <Test as Chainflip>::EpochInfo::epoch_index() + 1;
-		<VaultsPallet as VaultRotator>::keygen(
-			BTreeSet::from_iter(ALL_CANDIDATES.iter().cloned()),
-			rotation_epoch,
-		);
-		let keygen_ceremony_id = current_ceremony_id();
-
-		for p in ALL_CANDIDATES {
-			assert_ok!(VaultsPallet::report_keygen_outcome(
-				RuntimeOrigin::signed(*p),
-				keygen_ceremony_id,
-				Ok(NEW_AGG_PUB_KEY_PRE_HANDOVER)
-			));
-		}
-
-		VaultsPallet::on_initialize(1);
-		EthMockThresholdSigner::execute_signature_result_against_last_request(Ok(ETH_DUMMY_SIG));
-		assert!(matches!(
-			PendingVaultRotation::<Test, _>::get(),
-			Some(VaultRotationStatus::KeygenVerificationComplete { .. })
-		));
-
-		// Abort the vault rotation now
-		VaultsPallet::reset_vault_rotation();
-
-		assert!(PendingVaultRotation::<Test, _>::get().is_none());
-		assert_eq!(KeygenResolutionPendingSince::<Test, _>::get(), 0);
-		assert_eq!(VaultsPallet::status(), AsyncResult::Void);
-
-		// Can restart the vault rotation and succeed.
-		do_full_key_rotation();
-	});
-}
-
-#[test]
-fn can_recover_from_abort_vault_rotation_after_key_handover_failed() {
-	new_test_ext().execute_with(|| {
-		let rotation_epoch = <Test as Chainflip>::EpochInfo::epoch_index() + 1;
-		<VaultsPallet as VaultRotator>::keygen(
-			BTreeSet::from_iter(ALL_CANDIDATES.iter().cloned()),
-			rotation_epoch,
-		);
-		let keygen_ceremony_id = current_ceremony_id();
-		for p in ALL_CANDIDATES {
-			assert_ok!(VaultsPallet::report_keygen_outcome(
-				RuntimeOrigin::signed(*p),
-				keygen_ceremony_id,
-				Ok(NEW_AGG_PUB_KEY_PRE_HANDOVER)
-			));
-		}
-
-		VaultsPallet::on_initialize(1);
-		EthMockThresholdSigner::execute_signature_result_against_last_request(Ok(ETH_DUMMY_SIG));
-
-		// Key handover
-		const SHARING_PARTICIPANTS: [u64; 2] = [ALICE, BOB];
-		VaultsPallet::key_handover(
-			BTreeSet::from(SHARING_PARTICIPANTS),
-			BTreeSet::from_iter(ALL_CANDIDATES.iter().cloned()),
-			rotation_epoch,
-		);
-
-		let handover_ceremony_id = current_ceremony_id();
-
-		for p in ALL_CANDIDATES {
-			assert_ok!(VaultsPallet::report_key_handover_outcome(
-				RuntimeOrigin::signed(*p),
-				handover_ceremony_id,
-				Err(Default::default())
-			));
-		}
-
-		VaultsPallet::on_initialize(2);
-		assert!(matches!(
-			PendingVaultRotation::<Test, _>::get(),
-			Some(VaultRotationStatus::KeyHandoverFailed { .. })
-		));
-
-		// Abort by resetting vault rotation state
-		VaultsPallet::reset_vault_rotation();
-
-		assert!(PendingVaultRotation::<Test, _>::get().is_none());
-		assert_eq!(KeygenResolutionPendingSince::<Test, _>::get(), 0);
-		assert_eq!(VaultsPallet::status(), AsyncResult::Void);
-
-		// Can restart the vault rotation and succeed.
-		do_full_key_rotation();
-	});
-}
+// add test for testing active from block functionality
```

Comment:
?

Reply:
wanted to add a test here since. added now!